ZFSin/zfs/lib/libspl/include/wosix.h

```diff
@@ -45,7 +45,7 @@ extern int wosix_close(int fd);
 extern int wosix_ioctl(int fd, unsigned long request, void *zc);
 extern int wosix_read(int fd, void *data, uint32_t len);
 extern int wosix_write(int fd, const void *data, uint32_t len);
-extern int wosix_isatty(int fd);
+extern int wosix_isatty(intptr_t fd);
```

Comment:
Which would defeat the purpose of a posix wrapper library taking posix "int fd" and handling it for the local platform. 
```
man isatty
     int
     isatty(int fd);
```